test/integration/replicaset/replicaset_test.go

```diff
@@ -607,7 +596,7 @@ func TestPodOrphaningAndAdoptionWhenLabelsChange(t *testing.T) {
 	rs := newRS("rs", ns.Name, 1)
 	rs.Spec.Selector.MatchLabels = labelMap
 	rs.Spec.Template.Labels = labelMap
-	createRSsPods(t, c, []*v1beta1.ReplicaSet{rs}, []*v1.Pod{}, ns.Name)
+	rss, _ := createRSsPods(t, c, []*v1beta1.ReplicaSet{rs}, []*v1.Pod{}, ns.Name)
```

Comment:
I changed `TestPodOrphaningAndAdoptionWhenLabelsChange` based on your feedback. Do you mind to have another review for only the test before i worked on other tests? 
I commented out most tests to facilitate testing of individual test (I use `bazel test //...`, but am still looking for the regex option that filters test name). Ignore the `/* */` sign and commented imports.